Screenbox/Pages/SettingsPage.xaml

```diff
@@ -286,6 +288,23 @@
                         <x:String>ThemeAuto</x:String>
                     </ComboBox>
                 </ctc:SettingsCard>
+                <ctc:SettingsCard
+                    Margin="{StaticResource SettingsCardMargin}"
+                    Description="{strings:Resources Key=SettingsLanguageSelectionDescription}"
+                    Header="{strings:Resources Key=SettingsLanguageSelectionHeader}"
+                    HeaderIcon="{ui:FontIcon Glyph=&#xF2B7;}">
+                    <ComboBox
+                        x:Name="LanguageComboBox"
+                        MinWidth="200"
```

Comment:
Do we need to make the MinWidth larger than that the rest? I guess it's to keep the list from changing width while scrolling.

Reply:
Oh no, if the ComboBox is smaller than one of the items, the list is offset to the side. It's weird. Making the ComboBox 200 pixels wide is wide enough to fit all the items.

Reply:
Oh right, I'm already "used" to it. Change the language to pt-PT and open the theme option ComboBox and _voilá_.